packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -281,11 +281,11 @@ function FontLibraryProvider( { children } ) {
 					sucessfullyInstalledFontFaces?.length > 0 ||
 					alreadyInstalledFontFaces?.length > 0
 				) {
-					fontFamilyToInstall.fontFace = [
+					installedFontFamily.fontFace = [
```

Comment:
```suggestion
					// Use font data from REST API not from client to ensure
					// correct font information is used.
					installedFontFamily.fontFace = [
```

Nit: might be worth clarifying.